Rubberduck.Parsing/VBA/COMReferenceManagerBase.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using NLog;
+using System.Collections.Concurrent;
+using Rubberduck.VBEditor;
+using System.Runtime.InteropServices;
+using System.IO;
+using Rubberduck.Parsing.ComReflection;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public abstract class COMReferenceManagerBase : ICOMReferenceManager
+    {
+        private const string rubberduckGUID = "{E07C841C-14B4-4890-83E9-8C80B06DD59D}";
```

Comment:
The problem why I did not switch from the hard-coded value to a reference to the repository holding the GUIDs is that said repository is in the `Rubberduck` assembly. We cannot reference that from here because that creates a circular reference. Since I have no experience with reflection so far, I do not know whether we can get it that way.